src/Modules/Filter.lua

```diff
@@ -128,6 +128,21 @@ function(item)
 	end
 end);
 
+-- UnavailablePersonalLoot
+DefineToggleFilter("UnavailablePersonalLoot", AccountFilters,
+function(item)
+
+	if not item.sourceID
+		or not item.sourceParent
+		or not item.sourceParent.questID then
```

Comment:
Is it your intention to only apply this filter in popout lists or the minilist? `sourceParent` is only created for groups which are copied from the Main list data into some other context, since they may inherit a new `parent` depending on where they are going.

Also if it should specifically be 'Quests' then we would want to check that the parent `.key` is also `"questID"` since other Things have questID but aren't specifically 'Quests' (like Rares or Treasures).

Reply:
Hm, I was only thinking the minilist, but honestly I'm not familiar enough with the addon to know all the places this might not want to be applicable.

I've added the `.key` check as well.